### DISCUSSION TITLE
Resolving the 20-minute update issue with Rainmeter

### DISCUSSION THREAD

**Discussion by Cariboudjan** [osm-search/Nominatim#3448] [General]
Hello. I found [this issue](https://github.com/rainmeter/rainmeter/issues/367#issue-2008403316) and the matter will be resolved shortly. Unfortunately we neglected to consider that a piece of code was parsing both the location and the weather data from Google's API every 20-minutes, when only the weather data was required. A permanent fix to this is incoming within the next 7 days.

If Rainmeter has been blocked as a result of this, I apologize and am performing the corrective action. Can we see Rainmeter unblocked after this change?

Thanks.

**Comment by lonvia** [comment]
Yes. Please link to the PR with the fix here, once it is merged.

**Reply by Cariboudjan** [reply]
Thank you.

**Reply by Cariboudjan** [reply]
> Yes. Please link to the PR with the fix here, once it is merged.

I've pushed an update to all users. The update will only ping Nominatim when the location is manually changed by the user, or when the program is rebooted. The program does not automatically update, requiring the user to acknowledge an update prompt. So the 20-minute ping issue may not immediately resolve itself, but should significantly reduce over the next week and effectively disappear over the next few months.

For testing purposes, could you please unblock the application? I'd like to test the process before I can officially confirm the fix has been made.

**Reply by lonvia** [reply]
The block should be gone now.

**Reply by Cariboudjan** [reply]
Thanks. I've included a UserAgent that includes Droptop to identify us in the future in case there are issues. The test seems to be working properly.

**Reply by Cariboudjan** [reply]
The update has now been live for 24 hours. I'm curious if there has been a reduction in the number of pings.

**Reply by lonvia** [reply]
It certainly hasn't attracted the attention of the sysadmin, so you should be good. ;)

More seriously, I see that you have added a github repo to report issues to. That'll do. I'll let you know if there are problems again. Thank you for taking care of fixing the problem.
